linux-armv7/Toolchain.cmake

```diff
@@ -1,15 +1,15 @@
 set(CMAKE_SYSTEM_NAME Linux)
 set(CMAKE_SYSTEM_VERSION 1)
+set(CMAKE_SYSTEM_PROCESSOR arm)
 
 set(cross_triple "arm-linux-gnueabihf")
 
 set(CMAKE_C_COMPILER /usr/bin/${cross_triple}-cc)
 set(CMAKE_CXX_COMPILER /usr/bin/${cross_triple}-c++)
 set(CMAKE_Fortran_COMPILER /usr/bin/${cross_triple}-gfortran)
 
-set(CMAKE_FIND_ROOT_PATH /usr/${cross_triple})
-set(CMAKE_FIND_ROOT_PATH_MODE_PROGRAM NEVER)
-set(CMAKE_FIND_ROOT_PATH_MODE_LIBRARY ONLY)
-set(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE ONLY)
+# Discard path returned by pkg-config and associated with HINTS in module
+# like FindOpenSSL.
+set(CMAKE_IGNORE_PATH /usr/lib/x86_64-linux-gnu/ /usr/lib/x86_64-linux-gnu/lib/)
```

Comment:
Nop. In the case of this toolchain, the libraries are all installed on the system and not in dedicated root. This can be confirmed by looking at the files available in `/usr/${cross_triple}`:

```
root@ebb068fcc4d6:/usr/arm-linux-gnueabihf# find .
.
./bin
./bin/strip
./bin/ar
./bin/ld
./bin/nm
./bin/ld.gold
./bin/as
./bin/objcopy
./bin/ranlib
./bin/ld.bfd
./bin/objdump
```